server/utils/s3.ts

```diff
@@ -1,110 +1,66 @@
-import crypto from "crypto";
 import util from "util";
 import AWS from "aws-sdk";
-import { addHours, format } from "date-fns";
 import fetch from "fetch-with-proxy";
 import { v4 as uuidv4 } from "uuid";
 import Logger from "@server/logging/logger";
 
+// backward compatibility
 const AWS_S3_ACCELERATE_URL = process.env.AWS_S3_ACCELERATE_URL;
+const AWS_S3_UPLOAD_BUCKET_URL = process.env.AWS_S3_UPLOAD_BUCKET_URL;
+const AWS_S3_UPLOAD_BUCKET_NAME = process.env.AWS_S3_UPLOAD_BUCKET_NAME;
+const AWS_S3_FORCE_PATH_STYLE = process.env.AWS_S3_FORCE_PATH_STYLE;
```

Comment:
Thoughts on logging out a warning of deprecation and future removal if these are used?